packages/client/hmi-client/src/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue

```diff
@@ -205,7 +207,7 @@ const MINBARLENGTH = 1;
 const showSpinner = ref(false);
 
 const listModelLabels = ref<string[]>([]);
-const ensembleCalibrationMode = ref<string>(EnsembleCalibrationMode.EQUALWEIGHTS);
+const ensembleWeightsMode = ref<string>(EnsembleWeightsMode.EQUALWEIGHTS);
```

Comment:
Don't type this and it'll be autoset to `EnsembleWeightsMode` type

Reply:
My mistake, thank you for catching this